packages/auto-spies-core/src/errors/error-handling.spec.ts

```diff
@@ -2,48 +2,57 @@ import { errorHandler } from './error-handler';
 
 describe('throwArgumentsError', () => {
   let fakeExpectedArgs: any[];
+  let fakeFunctionName: string;
   let fakeActualArgs: any[];
   let actualError: any;
 
-  Given(() => {
-    fakeExpectedArgs = [1, 2];
-  });
-
-  When(() => {
-    try {
-      errorHandler.throwArgumentsError(fakeActualArgs);
-    } catch (error) {
-      actualError = error.message;
-    }
-  });
-
-  describe('Given actual args are empty and do not match', () => {
+  describe('Given a function name - ', () => {
```

Comment:
Let's use the captial letters `GIVEN` as a convention (to be consistent with the rest of the tests in this monorepo)